### PR TITLE
[Fix #115] Support elasticsearch-dsl > 6.1.0

### LINKED ISSUE #115: Support elasticsearch-dsl > 6.1.0
I am not sure about this issue but `django-elasticsearch-dsl` does not work with `elasticsearch-dsl` versions higher than 6.1.0, it shows an import error for `DocType` and `DocTypeMeta`, apparently they were removed in newer versions of `elasticsearch-dsl`.

### PR DESCRIPTION
**Pull request by safwanrahman**
~~**This does not work!**~~
Its something like verification purpose. The test are not passing!
need documentation fix and others.
Do not try this still!

this fixes #115

### CODE REVIEW

**Comment by sabricot** [comment]
Thank you @safwanrahman for your contribution i didn't have yet looked at it, but i promise i will soon.

File: django_elasticsearch_dsl/registries.py [hunk 1]
```diff
@@ -96,6 +100,11 @@ def update_related(self, instance, **kwargs):
         if not DEDConfig.autosync_enabled():
             return
 
+        # print(list(self._get_related_doc(instance)))
+        # import pdb
+        # pdb.set_trace()
+        print(instance.__class__, "ffff")
```
Comment (kornikopic) [inline django_elasticsearch_dsl/registries.py]: I think you forgot to remove your traces

File: django_elasticsearch_dsl/registries.py [hunk 2]
```diff
@@ -26,10 +26,14 @@ def register(self, index, doc_class):
             self._related_models[related].add(doc_class.django.model)
 
         for idx, docs in iteritems(self._indices):
+            # import pdb
+            # pdb.set_trace()
             if index._name == idx._name:
                 docs.add(doc_class)
                 return
 
+        print(self._indices, index)
```
Comment (kornikopic) [inline django_elasticsearch_dsl/registries.py]: You forgot to remove your debug

File: django_elasticsearch_dsl/registries.py [hunk 3]
```diff
@@ -26,10 +26,14 @@ def register(self, index, doc_class):
             self._related_models[related].add(doc_class.django.model)
 
         for idx, docs in iteritems(self._indices):
+            # import pdb
+            # pdb.set_trace()
```
Comment (kornikopic) [inline django_elasticsearch_dsl/registries.py]: I guess you forgot?

**Comment by safwanrahman** [comment]
@kornikopic Thanks for going through the changes. I have been working on this PR and still not finished.
I will remove the debugging codes when I finish the PR, but for debugging purpose, its necessary actually.

**Comment by safwanrahman** [comment]
This PR depends on https://github.com/elastic/elasticsearch-dsl-py/pull/1099

**Comment by safwanrahman** [comment]
After all, it passed all the tests and working perfectly! 💯 
Now needs couple of cleanup and a rebase and it will be ready for review! 🚀 
Moreover, need some documentation also!

@sabricot can you take a look into it?

**Comment by SeguinBe** [comment]
Is this going to be merged eventually?

**Comment by safwanrahman** [comment]
> Is this going to be merged eventually?

@SeguinBe This was blocked for https://github.com/elastic/elasticsearch-dsl-py/pull/1099. As the blocking PR has been merged recently, I think we can merge this PR soon.
Anyone is up there for reviewing the PR? @sabricot will you have some time?

**Comment by safwanrahman** [comment]
I can not find any people to review this PR. So going to merge it!

**Pull request merged**